src/Websites/Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -158,7 +161,8 @@ public override void ExecuteCmdlet()
                             Use32BitWorkerProcess =
                                 parameters.Contains("Use32BitWorkerProcess") ? (bool?)Use32BitWorkerProcess : null,
                             AutoSwapSlotName = parameters.Contains("AutoSwapSlotName") ? AutoSwapSlotName : null,
-                            NumberOfWorkers = parameters.Contains("NumberOfWorkers") ? NumberOfWorkers : WebApp.SiteConfig.NumberOfWorkers
+                            NumberOfWorkers = parameters.Contains("NumberOfWorkers") ? NumberOfWorkers : WebApp.SiteConfig.NumberOfWorkers,
+                            FtpsState = parameters.Contains("FtpsState") ? FtpsState : null
```

Comment:
If the default value is `AllAllowed`, shouldn't we use it instead of `null`?

Reply:
Yes we can use "AllAllowed" also